core/src/test/java/bisq/core/user/PreferencesTest.java

```diff
@@ -124,7 +124,7 @@ public void testGetUniqueListOfCryptoCurrencies() {
 
         preferences.readPersisted();
 
-        assertEquals(13, preferences.getCryptoCurrenciesAsObservable().size());
+        assertEquals(14, preferences.getCryptoCurrenciesAsObservable().size());
```

Comment:
Do not touch this file

Reply:
Why? If I do not modify this value build will fail:

Without changes:

```
$ ./gradlew build
........
bisq.core.user.PreferencesTest > testGetUniqueListOfCryptoCurrencies FAILED
    java.lang.AssertionError at PreferencesTest.java:127
14:50:14.799 [OpenOfferManager.ShutDownHook] DEBUG bisq.core.offer.OpenOfferManager - remove all open offers at shutDown
14:50:14.799 [OpenOfferManager.ShutDownHook] DEBUG bisq.core.offer.OpenOfferManager - remove all open offers at shutDown
14:50:14.799 [OpenOfferManager.ShutDownHook] DEBUG bisq.core.offer.OpenOfferManager - remove all open offers at shutDown

95 tests completed, 1 failed, 6 skipped

> Task :core:test FAILED

FAILURE: Build failed with an exception.
........
BUILD FAILED in 1m 43s
24 actionable tasks: 24 executed
```

After changing it:

```
$ ./gradlew build
........
BUILD SUCCESSFUL in 2m 1s
74 actionable tasks: 52 executed, 22 up-to-date
```